packages/core/src/demo/echo.js

```diff
@@ -41,6 +41,15 @@ const messageEchoCache = {};
   const apiClient = new APIClient({store: engine, urls: backend});
   const account = new Account(apiClient);
 
+  const sendConfirmation = async (messageId, conversationId, messageTimer, userId) => {
```

Comment:
I still don't think this is the appropriate name. Confirmation is used within the scope of the app to confirm the delivery of the message.

What you are doing here is a message deletion after ephemeral timer run out. Confirmation is not the correct term to be used.